src/app/helptext/storage/volumes/volume-status.ts

```diff
@@ -0,0 +1,73 @@
+import { T } from '../../../translate-marker';
+
+export default {
+    dialogFormFields: {
+        disk: {
+            placeholder: T('Member disk'),
+            tooltip: T(''),
+        },
+        passphrase: {
+            placeholder: T('Passphrase'),
+            tooltip: T('')
```

Comment:
Enter the current passphrase for the encrypted pool. This will be used to encrypt the new disk and integrate it into the pool.